dsp/modules/google.py

```diff
@@ -8,7 +8,8 @@
     import google.generativeai as genai
 except ImportError:
     google_api_error = Exception
-    print("Not loading Google because it is not installed.")
+    # print("Not loading Google because it is not installed.")
```

Comment:
Why commenting this out?

Reply:
If it stays, we print this every time anybody imports dspy, regardless of whether they are importing Google specifically. It's similar to how imports are managed on the Cohere side in: dsp/modules/cohere.py.

I imagine there may be a cleaner way throughout the project to manage optional dependencies.